packages/cli/package.json

```diff
@@ -9,10 +9,10 @@
   },
   "bugs": "https://github.com/Meeco/cli/issues",
   "dependencies": {
-    "@meeco/cryppo": "^0.7.0",
-    "@meeco/keystore-api-sdk": "^0.14.1-beta.3",
-    "@meeco/sdk": "^0.0.12-alpha.2",
-    "@meeco/vault-api-sdk": "^0.19.1-beta.4",
+    "@meeco/cryppo": "^0.7.1",
+    "@meeco/keystore-api-sdk": "0.14.1-beta.1",
+    "@meeco/sdk": "0.0.12-alpha.2",
+    "@meeco/vault-api-sdk": "0.19.1-beta.3",
```

Comment:
Any particular reason these got downgraded?

Reply:
not really, I think it was reverted during the merge. I will fix it.

> Any particular reason these got downgraded?